lasagne/init.py

```diff
@@ -1,5 +1,11 @@
 """
-Functions to create initializers for parameter variables
+Functions to create initializers for parameter variables.
+
+Usage
+-------
+>>> from lasagne.layers import DenseLayer
+>>> from lasagne.init import Constant, Glorot
+>>> l1 = DenseLayer((100,20), num_units=50, W=Glorot(), b=Constant(0.0))
```

Comment:
I'll give it a try. I did manage to squash commits based on your previous suggestions.